### ISSUE TITLE
Create Test model script for workshop

### ISSUE THREAD

**Issue by tomassosorio** [tomassosorio/DeepLearning#17]

**Comment by tomassosorio** [comment]
@miguelriemoliveira 
TODO:
import libs
import function with model

https://keras.io/models/model/#predict   ===> search predict

Important Cautions: when gives the predictions it will have the same nomenclature ( [Cat confidence, Dog confidence] ) has the input y label. Compare it with the real label and calculate precision factors.

**Comment by miguelriemoliveira** [comment]
Hi @tomassosorio 

is this the issue about me making changes in the code? If so please assign to me.

**Comment by miguelriemoliveira** [comment]
Sorry, now I saw you mentioned me, so this is my issue clearly. I will assign to me.

**Comment by miguelriemoliveira** [comment]
Hello @tomassosorio ,

I was supposed to create a test script and to change the TFRRecordfile.py

https://github.com/tomassosorio/DeepLearning/blob/master/deal_data/generate_TFRecordfile.py

but I saw you already made changes. It looks better now.

I will start with looking into the TRFRecordfile.py and only then I will work on the test script, so I am not sure if I will have the test script ready for our next meeting.

**Comment by tomassosorio** [comment]
Hi @miguelriemoliveira,
TFRecordfile is meant to the object detection, not for testing the models.
If you want to look at a test script you should look at:
https://github.com/tomassosorio/TOclassify/blob/master/TO_Test.py

**Comment by miguelriemoliveira** [comment]
I confused the file. I will review the one you mentioned

Em 26/10/2017 4:05 da tarde, "Tomás Osório" <notifications@github.com>
escreveu:

**Comment by miguelriemoliveira** [comment]
For now I am trying to work on #18 before going into this one...
